module/zfs/ddt.c

```diff
@@ -575,9 +596,51 @@ ddt_remove(ddt_t *ddt, ddt_entry_t *dde)
 	ddt_free(dde);
 }
 
+/*
+ * Check if the DDT is over its quota.  This can be due to a few conditions:
+ *   1. 'dedup_table_quota' is 0 and ddt_get_ddt_dsize exceeds this limit
```

Comment:
`'dedup_table_quota' is > 0`?

Reply:
ah, it should say "'dedup_table_quota' property is not 0 (none)..."  Fixed